platform/admin/ui/src/main/webapp/templates/installer/finish.handlebars

```diff
@@ -26,14 +26,12 @@
             <br/>
             <br/>
```

Comment:
❓ Is a restart required all of the time now? This makes is sound like a restart is only required if something is modified.